tests/unit/test.program.js

```diff
@@ -434,11 +434,10 @@ describe('program.defaultVersionGetter', () => {
       });
   });
 
-  it('returns git commit information in development', () => {
+  it_skipOnWin32('returns git commit information in development', () => {
```

Comment:
I'm pretty sure the `git-rev-sync` issue is not a Windows problem but an Appveyor one. I don't recall any mention of `git-rev-sync` problems in https://github.com/mozilla/web-ext/pull/751 .

Thus, could you make this skip happen only when a custom Appveyor env var is present? 

Also, I find the `it` wrapper hard to read. If it were done with an if statement inline it's only two extra lines. That would be tolerable to me. Something like:

````javascript
it('returns git commit information in development', () => {
  if (runningOnAppveyor()) {
    return this.skip();
  }
});
````
or maybe even without a function:

````javascript
it('returns git commit information in development', () => {
  if (process.env.APPVEYOR_CI) {
    return this.skip();
  }
});
````